packages/libs/coreui/src/components/inputs/SelectList.tsx

```diff
@@ -15,7 +18,7 @@ export interface SelectListProps<T> extends CheckboxListProps<T> {
   instantUpdate?: boolean;
 }
 
-export default function SelectList<T>({
+export default function SelectList<T extends ReactNode>({
```

Comment:
This seems strange to me. `T` is the type of the value. I don't think we ever want that to be a ReactNode. I think we should have `T extends string`, to be honest.

The change you made for `buttonDisplayContent` is perfect.

Reply:
Thanks. I'll try that, but it was my understanding that T needed to be something "compatible with" ReactNode.

Reply:
I think of `value` as what would be included in a form request, which is always a string. A number could also work, but I'm not convinced we need to do that.

This would be a breaking change, but I'm not sure if it would actually break anything.

Reply:
I've fixed it as you suggested. If it's a breaking change, it hasn't broken anything within this repo at least!